src/docs/asciidoc/user-guide/assertj-core-assertions-guide.adoc

```diff
@@ -654,7 +654,7 @@ The javadoc for `CharSequence` assertions is available https://www.javadoc.io/st
 [[assertj-core-iterable-assertions]]
 ==== Iterable/Collection/array assertions (WIP)
 
-TODO
+TODO(see also https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#extracted-properties-assertion[the old document])
```

Comment:
what would be great is to start filling this section with all the following sections from the old site
Iterable and arrays assertions :

- Combining filtering and assertions on iterables or arrays
- Assertions on extracted properties/fields of iterable/array elements
- Flat(map) extracting
- Assertions on results of a method call on iterable/array elements